src/server/dflycmd.cc

```diff
@@ -163,11 +163,33 @@ void DflyCmd::Run(CmdArgList args, ConnectionContext* cntx) {
     return ReplicaOffset(args, cntx);
   }
 
-  if (sub_cmd == "LOAD" && args.size() == 2) {
-    DebugCmd debug_cmd{sf_, cntx};
-    debug_cmd.Load(ArgS(args, 1));
-    return;
+  if (sub_cmd == "LOAD") {
+    return Load(args, cntx);
+  }
+
+  if (sub_cmd == "HELP") {
+    string_view help_arr[] = {
```

Comment:
It strange for me that help returns array, please consider to use R"()" string literal

Reply:
It's a usual thing to return an array for commands in valkey

Reply:
Indeed, as a human user it's easier to see errors like this than "line1\nline2" etc